### PR TITLE
Moved Voting-Logic to the core

### PR DESCRIPTION
**Pull request by sleeping-dev**
I moved the Voting-Logic to the core, combined Comment- and Combo Voting into one Controller and made it so that voting works reddit-style where you can up- or downvote as well as 'delete' your vote by clicking the same vote-arrow twice.

### CODE REVIEW

File: Smash_Combos.Core/Cqrs/Votes/PostComboVote/PostComboVoteRequest.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace Smash_Combos.Core.Cqrs.Votes.PostComboVote
+{
+    public class PostComboVoteRequest : IRequest<PostComboVoteResponse>
+    {
+        public int ComboId { get; set; }
+        public int CurrentUserId { get; set; }
+        public string UpOrDown { get; set; }
```
Comment (skittlz444) [inline Smash_Combos.Core/Cqrs/Votes/PostComboVote/PostComboVoteRequest.cs:12]: This sort of thing should be a bool in the core, you can convert it from string in the mapping profile or controller action.

**Review by skittlz444 — COMMENTED** [review]
Just change the up/downvote property in the core requests to a boolean (IsUpVote?) and have the controller action convert the string to bool. Keeps the core request clear of the framework style logic a bit more.

**Comment by Breadkenty** [comment]
@skittlz444 I made the change you requested. Let me know how it looks.

**Review by skittlz444 — COMMENTED** [review]
@Breadkenty is it possible to change the database to use a bool rather than a string for up/down? (mssql is type bit, can't remember for postgresql)

**Comment by Breadkenty** [comment]
@skittlz444 Yeah, we would just have to make a new migration once updating the `ComboVote` and `CommentVote` models. I think I'll let @sleeping-dev take that whenever he reviews my last commit.

**Comment by sleeping-dev** [comment]
I'll take care of that in a bit.

**Comment by sleeping-dev** [comment]
Looks good! I'll go ahead and merge it, we're launching baby!!

**Pull request merged**